frontend/src/utils/helpers/menuHelpers/setMenuToggle.tsx

```diff
@@ -0,0 +1,14 @@
+import Cookies from 'js-cookie';
+
+import { MENU_VALUES } from 'constants/menuPaths';
+
+export const setMenuToggle = (): boolean | undefined => {
+  const menuStatus: string | undefined = Cookies.get(MENU_VALUES.menuStatus);
+  const isMenuOpen = true;
+  if (!menuStatus || menuStatus === 'true') {
+    Cookies.set(MENU_VALUES.menuStatus, 'false');
+    return !isMenuOpen;
```

Comment:
It would be better to set isMenuOpen=false instead of return !isMenuOpen, so that we would have only 1 return in function.

Reply:
This Func should have 2 returns, because setMenuToggle must handle 2 conditions: 1. Control page reload and save last menu tab position after reload (its work with Cookies, if we update page, all info about menu position we will get from Cookie) 2. We should have returns case each time when user clicks on some menu tab without reload (for example page already loaded and we searching smth on tabs) we well use boolean status returns for this - we dnt use Cookies(but update its!) case handleSqueeze function on AuthorizedLayoutContainer cant handle set and get Cookie at the same time.